packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -114,10 +112,58 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
               data-ui-id="meeting-composite-people-button"
               disabled={props.disableButtonsForLobbyPage}
               label={meetingStrings.peopleButtonLabel}
+              styles={commonButtonStyles}
+            />
+          )}
+          {isEnabled(meetingCallControlOptions?.chatButton) !== false && (
+            <ChatButton
+              checked={props.chatButtonChecked}
+              showLabel={true}
+              onClick={props.onChatButtonClicked}
+              data-ui-id="meeting-composite-chat-button"
+              disabled={props.disableButtonsForLobbyPage}
+              label={meetingStrings.chatButtonLabel}
+              styles={commonButtonStyles}
             />
           )}
-        </Stack.Item>
+        </Stack>
       )}
     </Stack>
   );
 };
+
+const buttonContainerStyle: IStyle = {
+  padding: '0.75rem',
+  columnGap: '0.5rem'
```

Comment:
nit: does it look bad with slightly more space between the label and buttons? current (left) looks a little tighter spaced than the figma (right):
![image](https://user-images.githubusercontent.com/2684369/151629054-e132fcab-bfcf-4a76-aa1e-9838ec3d24a3.png)

Reply:
Not sure how much is padding and how much is columnGap, but maybe 0.625rem for column gap? 
![image](https://user-images.githubusercontent.com/2684369/151629225-a99563b4-0d62-4736-b4c8-c1b8d617745a.png)

Reply:
I haven't modified the default `IconButton` gaps here. Not sure I want to mess with that.